### DISCUSSION TITLE
Steam Deck!

### DISCUSSION THREAD

**Discussion by MrBrixican** [FosterFramework/Foster#22] [Show and tell]
Confirmed to be working on Steam Deck! Pretty decent performance as well, 120k (130k with custom renderer) frogs on Froggymark @ 60 FPS.

As an added flex, I even built it on the Steam Deck itself via dotnet cli.

![image](https://github.com/NoelFB/Foster/assets/23103373/f7e7ecc3-458f-4425-af02-540bc1dd295c)

**Comment by NoelFB** [comment]
Dang nice, that's awesome :D Glad it works so smoothly!!

**Comment by landon-codes** [comment]
Amazing!
I found Foster right after trying MonoGame. I'm wishing I would have found Foster first because I really want to try it.
